tools/ci_build/github/azure-pipelines/nuget/templates/test_win.yml

```diff
@@ -35,7 +35,7 @@ jobs:
   - task: BatchScript@1
     displayName: 'setup env'
     inputs:
-      filename: '$(Build.SourcesDirectory)\tools\ci_build\github\windows\setup_env_cuda_11.bat'
+      filename: '$(Build.SourcesDirectory)\tools\ci_build\github\windows\setup_env_gpu.bat'
```

Comment:
Make it as a template argument. 
Then call this file 3 times with:
1. setup_env.bat (which doesn't have any CUDA DLLs)
2. setup_env_cuda_11.bat  (which has CUDA only)
3. setup_env_gpu.bat (which has TRT and CUDA)